content/CORE/CORETutorials/Storage/Disks/DiskReplace.md

```diff
@@ -79,3 +79,65 @@ For pools with large amounts of data, this can take a long time.
 When the resilver is complete, the pool status screen updates to show the new disk and the pool status returns to **Online**.
 
 ![Storage Pools Status Replace Complete](/images/CORE/12.0/StoragePoolsStatusReplaceComplete.png "Replacement Complete")
+
+## CLI Method
```

Comment:
We don't really wanted to advertise this as the CLI method. This is a workaround that will only work for 13.0-RELEASE, nothing else.